lib/galaxy/datatypes/binary.py

```diff
@@ -169,6 +169,11 @@ def _get_samtools_version( self ):
                 break
         return version
 
+    @staticmethod
+    def merge(split_files, output_file):
+        params = ["samtools", "merge", "-f", output_file, " ".join(split_files)]
```

Comment:
Can you place the split_files in the ``params``  array and disable ``shell=True``.